main/src/ca/uwaterloo/flix/language/fmt/FormatType.scala

```diff
@@ -337,6 +337,7 @@ object FormatType {
           case Kind.RecordRow => "r" + id
           case Kind.SchemaRow => "s" + id
           case Kind.Predicate => "'" + id.toString
+          case Kind.JvmConstructorOrMethod => id.toString // to check
```

Comment:
@mlutze ?

Reply:
Let's add a tag `j`

Reply:
@chanattan Will you add a "j" in front of the string. Thanks!